### DISCUSSION TITLE
Monterey Guidance Revision 1

### DISCUSSION THREAD

**Discussion by robertgendler** [usnistgov/macos_security#103] [General]
Included in this release are guidance documents (HTML, PDF, XLS, SCAP) for the NIST SP 800-53r5 Low, Moderate, and High, CNSSI-1253, and CIS Critical Security Controls Version 8 baselines for macOS Monterey (12.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/macos_security/releases/tag/monterey_rev1'>Monterey Guidance Revision 1</a>.</em>

**Comment by grismemj** [comment]
DISA finally released the Monterey STIG today!

**Reply by robertgendler** [reply]
Only 106 days after our release :)

**Reply by grismemj** [reply]
Will the Monterey STIG baseline be released soon?

**Reply by robertgendler** [reply]
It's currently available in dev_monty. We need to clean up some things but plan on doing a release sometime soon.

**Reply by bernstei** [reply]
Not want to push, just curious, do you have any sort of estimate of the time scale?  Weeks?  Months?

**Reply by robertgendler** [reply]
Very very soon. Days, not weeks.

**Reply by bernstei** [reply]
Hi @robertgendler - any more updates on the Monterey STIG?

**Reply by robertgendler** [reply]
@bernstei our release 2 weeks ago included the STIG for monterey.

**Reply by bernstei** [reply]
Great - I didn't realize that, and was just monitoring this thread.  Thanks.

**Comment by grismemj** [comment]
Thanks for the update!

From: Bob Gendler ***@***.***>
Date: Monday, February 28, 2022 at 9:23 AM
To: usnistgov/macos_security ***@***.***>
Cc: GRISMER, MATTHEW J CIV USAF AFMC AFRL/RQVC ***@***.***>, Comment ***@***.***>
Subject: [URL Verdict: Neutral][Non-DoD Source] Re: [usnistgov/macos_security] Monterey Guidance Revision 1 (Discussion #103)

It's currently available in dev_monty. We need to clean up some things but plan on doing a release sometime soon.

—
Reply to this email directly, view it on GitHub<https://github.com/usnistgov/macos_security/discussions/103#discussioncomment-2266094>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AW6OSZOTDEET3S7DSARKF73U5OAOBANCNFSM5GLYEQ7A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
